### DISCUSSION TITLE
Register an AttributeConverter for a Type

### DISCUSSION THREAD

**Discussion by edeesis** [micronaut-projects/micronaut-data#1977] [Q&A] [answered]
Is there a way to register an AttributeConverter for a type? In the docs, it says if you don't own the type, you can put the `@TypeDef` on the field, but I use this type in a bunch of places, and would like to register the AttributeConverter (or even better, simply the `@TypeDef`) once.

**Comment by dstepanov** [comment, marked as answer]
That is not supported. Please create an issue. I think we can support default converters.

**Reply by edeesis** [reply]
https://github.com/micronaut-projects/micronaut-data/issues/1986 Thanks Denis!
